django/db/backends/postgresql/base.py

```diff
@@ -256,7 +255,9 @@ def get_database_version(self):
     def get_connection_params(self):
         settings_dict = self.settings_dict
         # None may be used to connect to the default 'postgres' db
-        if settings_dict["NAME"] == "" and not settings_dict["OPTIONS"].get("service"):
+        if settings_dict["NAME"] == "" and not settings_dict.get("OPTIONS", {}).get(
```

Comment:
I explicitly checked that we always configure "OPTIONS", when might it be missing?

Reply:
Sure, maybe it's not needed. However, it has nothing to do with connection pooling, so we should merge it as a separate cleanup.

Reply:
#17900